.changeset/gorgeous-hairs-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/site-kit': minor
+---
+
+(feat) Sections in context menu
```

Comment:
we've been using `feat:` in svelte and sveltekit with the sentence starting as a lowercase letter